docs/user-guide/how-to.md

```diff
@@ -123,6 +123,53 @@ jobs:
 Read more about [terraform-docs GitHub Action] and its configuration and
 examples.
 
+## Pre-commit Hook
+
+With [`pre-commit`], you can ensure your Terraform module documentation is kept
+up-to-date each time you make a commit.
+
+First, simply create or update a `.pre-commit-hooks.yaml`
+in the root of your Git repo with at least the following content:
+
+```yaml
+repos:
+  - repo: git://github.com/terraform-docs/terraform-docs
+    rev: <VERSION TAG OR SHA TO USE> # For example: "v0.11.2"
+    hooks:
+      - id: terraform-docs
+        args: [<ARGS TO PASS INCLUDING PATH>]  # For example: ["--output-file", "README.md", "./mymodule/path"]
+```
+
+(You can also include more than one entry under `hooks:` to update multiple docs. Just be sure
```

Comment:
Nit: try to keep the line length roughly at 80 characters, For example `Just be sure` should be moved to next line and the next line gets adjusted accordingly, if need be.